netbox/templates/dcim/device/interfaces.html

```diff
@@ -30,3 +30,29 @@
         </div>
     {% endif %}
 {% endblock bulk_extra_controls %}
+
+{% block head %}
+    {{ block.super }}
+    <style>
+        {% for status, color in table.Meta.cable_status_styles %}
+            tr[data-cable-status={{ status }}] {
+                background-color: var(--nbx-color-{{ color }}-a15);
+            }
+        {% endfor %}
+        tr[data-mark-connected=true] {
+            background-color: var(--nbx-color-success-a15);
+        }
+        tr[data-virtual=true] {
+            background-color: var(--nbx-color-primary-a15);
+        }
+        tr[data-enabled=disabled] {
+            background-color: var(--nbx-color-danger-a15);
+        }
+        tr[data-cable-status=connected] button.mark-installed {
+            display: none;
+        }
+        tr:not([data-cable-status=connected]) button.mark-planned {
+            display: none;
+        }
```

Comment:
This should be in the main scss file and not in the template

Reply:
Putting it in the main scss was not possible with this approach since the CSS needed to happen at run-time from my mistaken belief that the link statuses were user-customizable.

But since we've agreed that they're not, this can definitely go into the scss instead.